diff --git a/boost/regex/user.hpp b/boost/regex/user.hpp
index 96f7476..760ae8c 100644
--- a/boost/regex/user.hpp
+++ b/boost/regex/user.hpp
@@ -84,7 +84,7 @@
 // define this if you want to be able to access extended capture
 // information in your sub_match's (caution this will slow things
 // down quite a bit).
-// #define BOOST_REGEX_MATCH_EXTRA
+#define BOOST_REGEX_MATCH_EXTRA // enable regex captures
 
 // define this if you want to enable support for Unicode via ICU.
 // #define BOOST_HAS_ICU
